pyformat.py

```diff
@@ -160,6 +171,11 @@ def parse_args(argv):
                         help='drill down directories recursively')
     parser.add_argument('-a', '--aggressive', action='count', default=0,
                         help='use more aggressive formatters')
+    parser.add_argument('--remove-all-unused-imports', action='store_true',
+                        help='remove all unused imports, not just standard library \
+(requires "aggresive")')
```

Comment:
I try to avoid escaped newlines. Could you format that strings the way it is done [below](https://github.com/myint/pyformat/pull/6/files#diff-16bf71117e1d3562945ca16426f4faa4L164)?